code/__DEFINES/server_tools.dm

```diff
@@ -26,7 +26,7 @@
 #define SERVICE_CMD_PARAM_TARGET "target"
 #define SERVICE_CMD_PARAM_SENDER "sender"
 
-#define SERVICE_REQUEST_KILL_PROCESS "killme"
+//#define SERVICE_REQUEST_KILL_PROCESS "killme"	deprecated
```

Comment:
remove instead of commenting

Reply:
It's still supported on the tools side, this is there to make sure it isn't reused